tests/test_AudioLoading.cpp

```diff
@@ -0,0 +1,55 @@
+#include "test_Globals.hpp"
+
+#include <engine/GameWorld.hpp>
+
+BOOST_AUTO_TEST_SUITE(AudioLoadingTests)
+
+#if RW_TEST_WITH_DATA
```

Comment:
This needs to use the test predicate from #693

Reply:
done